packages/e2e-test-utils/package.json

```diff
@@ -39,7 +39,12 @@
 	},
 	"peerDependencies": {
 		"jest": ">=26",
-		"puppeteer": ">=1.19.0"
+		"puppeteer": ">=11"
+	},
+	"peerDependenciesMeta": {
+		"puppeteer": {
+			"optional": true
```

Comment:
Is this the default behavior in npm7 or are we trying to keep the current behavior when we switch to npm7

Reply:
Yes, it's to make sure it doens't install `puppeteer` in Gutenberg and in projects that use `@wordpress/e2e-test-utils` together with `@wordpress/scripts` that depends on `puppeteer-core`.